edge/pkg/devicetwin/dtcommon/util.go

```diff
@@ -52,3 +59,57 @@ func ValidateTwinValue(value string) bool {
 	match, _ := regexp.MatchString(pattern, value)
 	return match
 }
+
+func GetProtocolNameOfDevice(device *v1alpha2.Device) (string, error) {
+	protocol := device.Spec.Protocol
+	if protocol.OpcUA != nil {
+		return "OPCUA", nil
```

Comment:
I feel like these protocol names should have constants

Reply:
ok. i have moved the constants of protocol name from deviceontroller/downstream.go to devicecontroller/constants/device.go and make it a common reference.